NuGet/Microsoft.AspNet.SignalR.Client.Cpp.WinDesktop.targets.template

```diff
@@ -47,31 +47,27 @@
       <None Include="$(MSBuildThisFileDirectory)dll\x64\#Toolset#\Debug\signalrclient.dll">
         <DeploymentContent>true</DeploymentContent>
       </None>
-      <ReferenceCopyLocalPaths Include="$(MSBuildThisFileDirectory)dll\x64\#Toolset#\Debug\signalrclient.dll" />
     </ItemGroup>
     <ItemGroup Label="x64 and #Toolset# and Release" Condition="'$(Platform.ToLower())' == 'x64' And '$(PlatformToolset.ToLower())' == '#Toolset#' And $(Configuration.ToLower().IndexOf('debug')) == -1">
       <CopyToOutput Include="$(MSBuildThisFileDirectory)dll\x64\#Toolset#\Release\signalrclient.dll" />
       <CopyToOutput Include="$(MSBuildThisFileDirectory)dll\x64\#Toolset#\Release\signalrclient.pdb" />
       <None Include="$(MSBuildThisFileDirectory)dll\x64\#Toolset#\Release\signalrclient.dll">
         <DeploymentContent>true</DeploymentContent>
       </None>
-      <ReferenceCopyLocalPaths Include="$(MSBuildThisFileDirectory)dll\x64\#Toolset#\Debug\signalrclient.dll" />
     </ItemGroup>
     <ItemGroup Label="Win32 and #Toolset# and Debug" Condition="'$(Platform.ToLower())' == 'Win32' And '$(PlatformToolset.ToLower())' == '#Toolset#' And $(Configuration.ToLower().IndexOf('debug')) &gt; -1">
```

Comment:
wow.. good catch. Looks like the other bug concealed this one. I will try to see what happens and will update the PR...